assets/js/googlesitekit/notifications/components/common/CTALink.js

```diff
@@ -86,16 +106,28 @@ export default function CTALink( {
 		}
 	};
 
+	useEffect( () => {
+		return () => {
+			clearError( 'notificationCTAClick', [ id ] );
+		};
+	}, [ clearError, id ] );
+
 	return (
-		<SpinnerButton
-			className="googlesitekit-notification__cta"
-			href={ ctaLink }
-			onClick={ handleCTAClick }
-			disabled={ isAwaitingCTAResponse || isNavigatingToCTALink }
-			isSaving={ isAwaitingCTAResponse || isNavigatingToCTALink }
-		>
-			{ ctaLabel }
-		</SpinnerButton>
+		<Fragment>
+			{ showError && ctaError && (
+				<ErrorText message={ ctaError.message } />
```

Comment:
Perhaps we could create a NotificationErrorHandler component that calls `getError` and renders it as well as clears the error (moving the above `useEffect` call). Setting/receiving the error can still happen in the event handlers (onCTAClick / onDismiss) within the different actions components (CTALink, Dismiss, ..)

Reply:
Thanks @jimmymadon, that's a good idea. I've pushed an update with this change, although I have simply named the component `Error` as this seems more consistent when compared to the other notification components:

![image](https://github.com/user-attachments/assets/0d7460a2-98fd-41c5-a672-edad9db8ddd4)

Reply:
I've created a followup issue as per the above: https://github.com/google/site-kit-wp/issues/9925